AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+# AzTS Data Table
+
+
+## **Overview**
+
+Azure Tenant Security Solution centrally scans subscriptions and get control scan result, processed RBAC data, resource inventory data, subscription meta data etc. and push all the data into a blob storage of central storage account.
+
+## **AzTS Data Table details**
+
+| Table Name | Description |
+|--|--|
+| [RBAC data](Readme.md#rbac-data)  | Role Based Access Control (RBAC) contains all PIM, permanent and classic role assignments details. |
```

Comment:
Might not want to suffix all these with "data"
Just "RBAC", "Baseline Controls", sounds fine.
Look at these as data entities that don't need a suffix every time you mention it